receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -247,6 +248,12 @@ func (jr *jReceiver) startAgent(host component.Host) error {
 			return err
 		}
 		jr.agentProcessors = append(jr.agentProcessors, processor)
+
+		jr.settings.Logger.Info("Starting Binary Thrift server",
```

Comment:
I've follwed these field throughout while logging.
Do we need to log any other field as well here?

Reply:
```suggestion
		jr.settings.Logger.Info("Starting UDP server for Binary Thrift",
```